tools/README.md

```diff
@@ -0,0 +1,54 @@
+# Performance testing tool for ballerina-message-broker 
+
+This tool allows ballerina message broker developers to get an idea about the performance of the message broker after a modification is done in the code. Following criteria is going to be used in evaluating the performance.
```

Comment:
Should we say "Following criteria is used" instead of "going to be used"?

Reply:
Latency calculation is not yet implemented.That's why I used "going to be used" instead of "is used" . WDYT?

Reply:
Then let's remove the section about latency and do the change Maryam has suggested. When latency calculation is implemented, let's add that section.

Reply:
Removed latency parameter

Reply:
@hasithajayasundara Shall we add an issue to track this to make sure we don't lose track?